pwndbg/commands/context.py

```diff
@@ -162,11 +163,26 @@ def context_disasm():
 
     return banner + result
 
+# XXX: move this part to color/ just like color/disasm.py:instrution()?
```

Comment:
Yeah, it is okay to move it to color/disasm.py

Reply:
For me, it is connected to context. Actually, I would like to create a new file such as `color/syntax_highlight.py` and move all related things to there.

Reply:
I like the idea of `syntax_highlight.py` :+1:

Reply:
Then we need to decide which function need to move to there. You can give me some comments when you review the second time.